ui/tests/unit/machines/secrets-machine-test.js

```diff
@@ -524,10 +524,10 @@ module('Unit | Machine | secrets-machine', function() {
       event: 'CONTINUE',
       params: 'database',
       expectedResults: {
-        value: 'list',
+        value: 'details',
```

Comment:
curious why this changed 🤔

Reply:
I confirmed, it looks the same as the other tab-modified secret engines (e.g. SSH).  They had it set as the default and when we moved the tab structure this needed to be modified.  But I won't lie, that's about as far as I got with it.  I'll check it out more as I move onto the wizard ticket.